aspnetcore/getting-started/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Get started with ASP.NET Core
 author: rick-anderson
-description: A short tutorial that creates and runs a basic Hello World app using ASP.NET Core.
+description: A short tutorial using the DotNet CLI to create and run a basic Hello World app using ASP.NET Core.
```

Comment:
I think .NET CLI is more of a recognised term, but feel free to disregard if you've seen that DotNet CLI is a better fit here:

```suggestion
description: A short tutorial using the .NET CLI to create and run a basic Hello World app using ASP.NET Core.
```